crates/tests_integ/tests/sanity_checks.rs

```diff
@@ -819,17 +819,17 @@ async fn tests_integ_select_remote_function() -> errors::Result<()> {
     insert.commit().await?;
 
     drop(insert);
-    let dates = [
-        Utc.ymd(2010, 1, 1).and_hms(0, 0, 0),
-        Utc.ymd(2011, 2, 28).and_hms(0, 0, 0),
-        Utc.ymd(2012, 2, 29).and_hms(0, 0, 0),
+    let dates = vec![
```

Comment:
why you change a testcase for remote functions? why not add a new testcase?

Reply:
@jinmingjian  this test case isn't correct and has an impact on this PR.

Reply:
ok. In this case, it is better to make a change for this testcase and then add a new one